src/NzbDrone.Core/Indexers/Definitions/DesiTorrents.cs

```diff
@@ -56,25 +51,4 @@ protected override IndexerCapabilities SetCapabilities()
             return caps;
         }
     }
-
-    public class DesiTorrentsParser : Gazelle.GazelleParser
-    {
-        public DesiTorrentsParser(GazelleSettings settings, IndexerCapabilities capabilities)
-            : base(settings, capabilities)
-        {
-        }
-
-        public override IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
-        {
-            var releases = base.ParseResponse(indexerResponse);
-
-            foreach (TorrentInfo release in releases)
-            {
-                release.MinimumRatio = 0.6;
-                release.MinimumSeedTime = 259200;
```

Comment:
Might need to keep this as Unit3d standard is 1.0 and 172800

Reply:
Was working on that locally, but having issues with it accepting this.

didn't like torrent info / release info

also needed a change to the base to make it overidable

Reply:
Cool, ping me if you need any help